apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v2_0/config/KeystoneAuthenticationModule.java

```diff
@@ -234,6 +234,13 @@ public String get() {
       return authenticationMethods.get(credentialType);
    }
 
+   @Provides
+   @Singleton
+   @Named("credentialsRetryable")
+   protected final boolean credentialsRetryable(Function<Credentials, Access> getAccess) {
```

Comment:
Hmm... Nope, the code should call `credentialType.retryable()`, but it makes more sense to object a CredentialType object than a boolean named "something". The code is easier to read and understand if injecting the object (and easier to extend in the future).